packages/platform-test-suite/bin/test.sh

```diff
@@ -187,11 +185,21 @@ then
   cmd="${cmd} DPNS_TOP_LEVEL_IDENTITY_PRIVATE_KEY=${identity_private_key}"
 fi
 
+if [ -n "$faucet_wallet_use_storage" ]
+then
+  cmd="${cmd} FAUCET_WALLET_USE_STORAGE=${faucet_wallet_use_storage}"
+fi
+
+if [ -n "$faucet_wallet_storage_dir" ]
+then
+  cmd="${cmd} FAUCET_WALLET_STORAGE_DIR=${faucet_wallet_storage_dir}"
+fi
+
 if [ -n "$GITHUB_ACTIONS" ]
 then
   cmd="${cmd} NODE_ENV=test node_modules/.bin/mocha -b ${scope_dirs}"
 else
-  cmd="${cmd} NODE_ENV=test mocha -b ${scope_dirs}"
+  cmd="${cmd} NODE_ENV=test yarn mocha -b ${scope_dirs}"
```

Comment:
Not sure we install yarn to docker image where this script is using. I would keep npm.

Reply:
We can't use npm in script, because only yarn support workspace references, throwing me an
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:~
```

Also, we use yarn in the Dockerfile of platform test suite inside.

Reply:
Could you please tests that docker image with updated test helper works please?